xskillscore/core/contingency.py

```diff
@@ -0,0 +1,787 @@
+from functools import wraps
+
+import numpy as np
+import xarray as xr
+
+from xhistogram.xarray import histogram as xhist
+
+__all__ = ['histogram', 'Contingency']
+
+OBSERVATIONS_NAME = 'observations'
+FORECASTS_NAME = 'forecasts'
+
+
+def histogram(*args, bins=None, bin_names=None, **kwargs):
+    """
+        Wrapper on xhistogram to deal with Datasets appropriately
```

Comment:
I think for all these cases you want:

```python
"""Wrapper on xhistogram ...

"""
```

I.e., the description aligned right with the triple quotes. This makes sure that `sphinx` documentation parses the API correctly.